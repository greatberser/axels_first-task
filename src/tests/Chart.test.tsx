import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

describe('Chart', () => {
  it('should match a snapshot', () => {
    const { asFragment } = render(<div>Chart Component</div>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should display chart title', () => {
    render(<div>Chart Component</div>);
    expect(screen.getByText(/chart component/i)).toBeInTheDocument();
  });
});
