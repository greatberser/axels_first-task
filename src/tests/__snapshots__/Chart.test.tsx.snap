// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Chart should match a snapshot 1`] = `
<DocumentFragment>
  <div>
    Chart Component
  </div>
</DocumentFragment>
`;
